packages/webapp/src/containers/Documents/util.js

```diff
@@ -24,7 +24,7 @@ export function useStringFilteredDocuments(documents, filterString) {
         ?.toLowerCase()
         .normalize('NFD')
         .replace(/\p{Diacritic}/gu, '')
-        .replace(/\W/g, '')
+        .replace(/[^\p{L}0-9]/gu, '')
         .replace(/_/g, '')
```

Comment:
The new regex also replaces underscores (I learned this when I looked at your PR from yesterday)!

So the next line isn't necessary, but nor does it do anything. So I approve either way! 🙂

Reply:
Thank you Joyce!!! I didn't like it so I removed them.
It would be appreciated if you could re-review, thank you!!!